depends/packages/qt.mk

```diff
@@ -61,8 +52,6 @@ $(package)_config_opts += -no-sql-psql
 $(package)_config_opts += -no-sql-sqlite
 $(package)_config_opts += -no-sql-sqlite2
 $(package)_config_opts += -no-use-gold-linker
-$(package)_config_opts += -no-xinput2
-$(package)_config_opts += -no-xrender
```

Comment:
I believe we may have to specify some new gfx options, notice that "vnc" is now enabled for Linux.

Reply:
Usually this would be achieved by `no-gfx-vnc`, but Qt5.8 changed large parts of the configure. `-no-xinput2` still works, I'll add it back.